packages/components/src/ComboBox/docs/allowEmptyCollection.tsx

```diff
@@ -1,15 +1,12 @@
-import { ComboBox, ComboBoxOptions, Label } from "@hopper-ui/components";
+import { ComboBox } from "@hopper-ui/components";
 
 export default function Example() {
     return (
         <ComboBox
             allowsEmptyCollection
-            aria-label="pets"
+            label="Roles"
         >
-            <Label>Roles</Label>
-            <ComboBoxOptions>
-                {[]}
-            </ComboBoxOptions>
+            {[]}
```

Comment:
make the children optional. This synthax really isn't pretty

Reply:
can you check if we use that somewhere else aldo?

Reply:
We determined that this wouldn't actually happen in real life scenarios and it's just for documentation, so we won't change it.